Apto/Base/Client/backend/apto-base/actions/content-snippet.actions.js

```diff
@@ -40,11 +40,18 @@ const ContentSnippetActions = function(MessageBusFactory, PageHeaderActions) {
             let commandArguments = [];
             commandArguments.push(ContentSnippet.name);
             commandArguments.push(ContentSnippet.active);
-            commandArguments.push(ContentSnippet.content);
+
+            if (Object.values(ContentSnippet.content).some(x => (x !== null && x !== ''))) {
+                commandArguments.push(ContentSnippet.content);
+            } else {
+                ContentSnippet.content = {};
```

Comment:
we should not change origin value here, instead of we should just pass the empty object to the commandArguments